src/openkit-shared/Core/Caching/BeaconCacheEvictor.cs

```diff
@@ -19,6 +19,9 @@
 using Dynatrace.OpenKit.Providers;
 using System;
 using System.Threading;
+#if WINDOWS_UWP

```

Comment:
instead of wrapping the using with `#if`, please use the full namespace for `Task` class.